src/pages/index.ru.md

```diff
@@ -0,0 +1,76 @@
+---
+template: home-page
+path: /
+title: Home
+hero:
+  buttons:
+    - text: Get Email Updates
+      url: '/#mailing-list'
+    - text: Start Building
+      url: /developers
+  subtitle: Deposit and redeem BTC in DeFi without intermediaries.
+  title: The safe way to earn with your Bitcoin.
+features:
+  - description: 'Every TBTC is backed 1:1 with BTC'
+    title: Backed
+  - description: No intermediaries - Redeem TBTC for BTC at any time
+    title: Permissionless
+  - description: Audited and open-source
+    title: Secure
+  - description: 3 steps to convert from BTC to TBTC and back
+    title: Simple
+spotlight_1:
+  body: >-
+    tBTC will begin moving toward full mainnet launch over the coming weeks.
+    Launch will occur over a series of milestones starting April 24th.
+  button:
+    text: Read more
+    url: 'https://tbtc.network/news/2020-04-24-tbtc-launch-key-dates/'
+  label: Announcement
+  title: 'tBTC Launch: Key Dates'
+spotlight_2:
+  button:
+    text: Developer Toolkit
+    url: /developers
+  label: Developers
+  title: Integrate TBTC to add Bitcoin to your dApp
+integrations_section:
```

Comment:
Would it make sense to break this out to non-localized somehow? It's going to be frustrating to do this for each language we support.

Maybe our process should be deleting and regenerating pages that haven't been localized to make this a *little* better..?

Reply:
I'm down with moving this out of the home page config into something separate.

>Maybe our process should be deleting and regenerating pages that haven't been localized to make this a little better..?

Can you elaborate a little more? Not sure if I'm understanding correctly what you mean.

Reply:
Created #127!

Reply:
I just didn't want to edit this section `n` times for `n` languages- no need to regen with #127.